arch/sim/src/sim/up_simsmp.c

```diff
@@ -53,7 +54,11 @@
 struct sim_cpuinfo_s
 {
   int cpu;    /* CPU number */
+#ifdef CONFIG_HOST_MACOS
+  sem_t *done;
+#else
   sem_t done; /* For synchronization */
```

Comment:
should we switch to named semp for all platform? it's hard to maintain two codebase.

Reply:
I think it's a good idea